### PR TITLE
Create method for prevent sms on android and ios

### PR DESCRIPTION
**Pull request by gagahkharismanuary**
## What does this PR do?
Create method for prevent sms on android and ios

## Why are we doing this? Any context or related work?
- https://kitabisa.atlassian.net/browse/UPS-1861
- https://kitabisa.atlassian.net/browse/UPS-1908
- https://kitabisa.atlassian.net/browse/UPS-1906

### CODE REVIEW

File: ischecksms/is_check_sms.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package ischecksms
+
+// Prevent Sms For Android/IOS
+func isCheckSms(donationSource string) bool {
```
Comment (triardn) [inline ischecksms/is_check_sms.go:4]: @gagahkharismanuary function name nya mungkin bisa diubah jadi isSendingSMS pak biar lebih readable

**Review by triardn — CHANGES_REQUESTED** [review]

File: ischecksms/is_check_sms.go [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	return !(donationSource == "android" || donationSource == "ios")
```
Comment (ardityawahyu) [inline ischecksms/is_check_sms.go:5]: ini perlu di-lowering case biar lebih aman

Comment (gagahkharismanuary) [reply ischecksms/is_check_sms.go:5]: ini di lowering-case dari sisi code ? pake `strings.ToLower` maksdnya mas ? @ardityawahyu

Comment (ardityawahyu) [reply ischecksms/is_check_sms.go:5]: iya @gagahkharismanuary , sebelum dicocokin, lowering dulu.

**Comment by gagahkharismanuary** [comment]
@ardityawahyu @triardn updated ya 🙏

**Review by ardityawahyu — APPROVED** [review]

File: ischecksms/is_check_sms_test.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package ischecksms
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestIsCheckSmsIsSuccess(t *testing.T) {
```
Comment (triardn) [inline ischecksms/is_check_sms_test.go:9]: @gagahkharismanuary function ini nya gk disamain sama actual function name nya Gah? wkwk

**Review by triardn — CHANGES_REQUESTED** [review]

File: ischecksms/is_check_sms_test.go (hunk 3 above)
Comment (gagahkharismanuary) [reply ischecksms/is_check_sms_test.go:9]: jir lupa ganti wkwk, wait fixing @triardn

**Review by triardn — APPROVED** [review]

**Pull request merged**